plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Capabilities.js

```diff
@@ -279,9 +296,9 @@ Ext.define('NX.coreui.controller.Capabilities', {
 
     if (list) {
       me.getCapabilityTypeStore().load(
-        function() {
-          me.reselect();
-        }
+          function() {
```

Comment:
Found it: http://screencast.com/t/2uWRgBkl
And that is in out standard code style https://github.com/sonatype/codestyle/blob/master/sonatype-idea.xml